extensions.lua

```diff
@@ -152,6 +156,28 @@ local function process_config(conf)
         end
 
         for _, event in ipairs(fconf.events) do
+            if event.http == nil then
+                -- do nothing
+            elseif cartridge_args.http_enabled == false then
+                return nil, ExtensionConfigError:new(
+                    "Cartridge was inited with the HTTP server turned off. " ..
+                    "Set the option \"http_enabled\" to \"true\" value and apply config again."
+                )
```

Comment:
Does it mean if we have a single instance with HTTP disabled we can't apply config?

Reply:
???

Reply:
Are you kidding? Imagine the cluster with HTTP enabled on routers and disabled on storage.

Reply:
Done